### PR TITLE
EPMRPP-88609 || update events for Make decision bulk operations

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js [hunk 1]
```diff
@@ -46,50 +48,101 @@ const getOpenModalEventCreator = (place) => (defectFromTIGroup, actionPlace = ''
...
     defectFromTIGroup,
   );
-  const isBulkOperation = basicEventParameters.condition === 'bulk';
-  if (!isBulkOperation) {
-    basicEventParameters.type = hasSuggestions ? 'with_ml' : 'without_ml';
-  }
+
+  basicEventParameters.type = hasSuggestions
+    ? getTypePrefix('with_ml', issueActionType)
+    : getTypePrefix('without_ml', issueActionType);
```
Comment (AmsterGet) [inline app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js:84]: Can be simplified
```
basicEventParameters.type = getTypePrefix(hasSuggestions ? 'with_ml' : 'without_ml', issueActionType); 
```
or the function `getTypePrefix` can be removed at all.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js [hunk 2]
```diff
@@ -46,50 +48,101 @@ const getOpenModalEventCreator = (place) => (defectFromTIGroup, actionPlace = ''
...
     status,
     switcher,
     icon_name: iconName,
-    element_name: 'apply',
+    element_name: getElementName(issueActionType),
   };
 };
-const getClickOnApplyAndContinueBtnEventCreator = (place) => (
+
+const getSwitcher = (items, issueType) => {
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js]: Done.

File: app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js (hunk 1 above)
Comment (BlazarQSO) [reply app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js]: Done.

File: app/src/common/constants/actionTypes.js [hunk 3]
```diff
@@ -17,9 +17,9 @@
 export const START_LAUNCH = 'startLaunch';
 export const FINISH_LAUNCH = 'finishLaunch';
 export const DELETE_LAUNCH = 'deleteLaunch';
-export const POST_ISSUE = 'postIssue';
-export const LINK_ISSUE = 'linkIssue';
-export const UNLINK_ISSUE = 'unlinkIssue';
+export const POST_ISSUE = 'post_issue';
+export const LINK_ISSUE = 'link_issue';
+export const UNLINK_ISSUE = 'unlink_issue';
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/common/constants/actionTypes.js]: Done.

**Pull request merged**